moonshot/integrations/web_api/services/cookbook_service.py

```diff
@@ -1,7 +1,9 @@
+from moonshot.src.cookbooks.cookbook_arguments import CookbookArguments
```

Comment:
@imda-normanchia 
I think the name `CookbookArguments`  with the `Arguments` suffix doesn't fit our usage. Can alias to a more fitting name or maybe just `Cookbook`?
Same goes for the other ms lib imported types you just added